src/App.jsx

```diff
@@ -73,6 +73,26 @@ export function App({ gmApiKey }) {
     }
   }, [taps, setInitialLoad, initialLoad, setTaps]);
 
+  // Initial page load: get browser's default language and init localization
+  useEffect(() => {
+    const language = window.navigator.userLanguage || window.navigator.language;
+    const supportedLanguages = ["en", "ja"];
+
+    if (language.includes('en')){
+      setLocale("en")
+      console.log("en")
+    }
+    else if (language.includes('ja')){
+      setLocale("ja")
+    }
+    else{
+      // Default language is Japanese if browser's is something else
+      setLocale("ja")
+    }
```

Comment:
this could be shortened by just using the `else` statement without the `else if`. The logic is the same.